### PR TITLE
Kuzzle is disconnected

### PR DESCRIPTION
**Pull request by AnthonySendra**
* Implement special page when the BO can't connect to Kuzzle
* Implement modal with error message

### CODE REVIEW

File: src/components/Error/KuzzleDisconnectedPage.vue [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+</style>
+
+<script>
+  import kuzzle from '../../services/kuzzle'
+  import KuzzleDisconnected from './KuzzleDisconnected'
+  import { setConnection } from '../../vuex/modules/common/kuzzle/actions'
+
+  let idConnect
+  let idReconnect
```
Comment (jenow) [inline src/components/Error/KuzzleDisconnectedPage.vue]: Shouldn't this be in data ?

Comment (AnthonySendra) [inline src/components/Error/KuzzleDisconnectedPage.vue]: Nop, we don't have to bind something, nor to display these variables

**Comment by jenow** [comment]
+1

**Comment by stafyniaksacha** [comment]
+1

**Pull request merged**